app/models/user.rb

```diff
@@ -6,4 +6,9 @@ class User < ApplicationRecord
 
   validates_presence_of :name, :phone, :address
 
+  has_many :orders, class_name: 'Cartify::Order', foreign_key: :user_id
```

Comment:
update the user model specs to test these associations

Reply:
done